src/signer/src/bitcoin_api.rs

```diff
@@ -0,0 +1,19 @@
+use ic_cdk::api::management_canister::bitcoin::{
+    bitcoin_get_balance, BitcoinNetwork, GetBalanceRequest,
+};
+
+/// Returns the balance of the given bitcoin address.
+///
+/// Relies on the `bitcoin_get_balance` endpoint.
+/// See [Bitcoin API](https://internetcomputer.org/docs/current/references/ic-interface-spec/#ic-bitcoin_get_balance)
+pub async fn get_balance(network: BitcoinNetwork, address: String) -> u64 {
+    let min_confirmations = None;
+    let balance_res = bitcoin_get_balance(GetBalanceRequest {
+        address,
+        network,
+        min_confirmations,
+    })
+    .await;
+
+    balance_res.unwrap().0
+}
```

Comment:
Would it be possible to move this file into a directory dedicated to bitcoin?  Something like `sign/btc/`, to be consistent with `sign/eth/`.  We might want to rename `sign` at some point, it made sense when separating signing code from user profile code.  But we can stress about renaming another time. :-)